angular-client/src/components/battery/battery.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="battery">
+  <div class="battery-nub"></div>
+  <div class="battery-bar" [ngStyle]="{ 'background-color': item.backColor }">
+    <ng-template [ngIf]="item.full == false">
+      <div [ngStyle]="{ 'background-color': item.fillColor, height: item.height, width: '30px' }"></div>
```

Comment:
this should be height 100% the user should just be able to change how big the height and width the battery container is and it should expand to fit. shouldnt be hard coding heights and widths